.github/workflows/build.yml

```diff
@@ -48,37 +48,16 @@ jobs:
   # only build this on our default Java version but for all operating systems
   # we are supporting. This allows for a fast execution and fast feedback. The
   # extended tests will run later under certain conditions.
+  # If the OS is ubuntu, we also run the SonarCloud analysis.
   basic:
     runs-on: ${{ matrix.os }}-latest
     needs: validation
     timeout-minutes: 15
     strategy:
       matrix:
-        java: [ 8 ]
+        java: [ 11 ]
         os: [ubuntu, macos, windows]
     name: with Java ${{ matrix.java }} on ${{ matrix.os }}
-    steps:
-      - name: Check out repo
-        uses: actions/checkout@v2
-      - name: Set up Java
-        uses: actions/setup-java@v2
-        with:
-          java-version: ${{ matrix.java }}
-          distribution: temurin
-          cache: 'gradle'
-      - name: Gradle build
-        uses: gradle/gradle-build-action@v2
-        with:
-          arguments: --refresh-dependencies --stacktrace --scan clean build
-
-  # SonarCloud analysis does not support Java 8, hence we need to execute the
-  # analysis separately with Java 11. This is fine because we are done with the basic
-  # builds for the contributors.
-  sonarcloud:
```

Comment:
I wonder if we really want to move the SonarCloud analysis into the basic build. Before:

<img width="1516" alt="grafik" src="https://user-images.githubusercontent.com/11643714/158082987-8371c618-feea-4036-bec1-5cb0505828ec.png">

Now:

<img width="1515" alt="grafik" src="https://user-images.githubusercontent.com/11643714/158082993-250f4cfb-1d1b-4c55-828c-2f1f435bae93.png">

IIRC so far a failing SonarCloud analysis wouldn't stop following jobs, because it represents a separate branch in the workflow. This is now no longer the case I believe.

Reply:
Yeah, I don't mind. I think we can commit to a stricter workflow.